src/main/java/uk/gov/gchq/gaffer/doc/function/ToBytesExample.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.doc.function;
+
+import uk.gov.gchq.koryphe.impl.function.ToBytes;
+
+import java.nio.charset.StandardCharsets;
+
+public class ToBytesExample extends FunctionExample {
+    public static void main(final String[] args) {
+        new ToBytesExample().runAndPrint();
+    }
+
+    public ToBytesExample() {
+        super(ToBytes.class);
+    }
+
+    @Override
+    protected void runExamples() {
+        stringToBytes();
+    }
+
+    private void stringToBytes() {
+        // ---------------------------------------------------------
+        final ToBytes toBytes = new ToBytes(StandardCharsets.UTF_16);
+        // ---------------------------------------------------------
+
+        runExample(toBytes, null, "example String", "", null, 1);
```

Comment:
Is there anything that can be done to make the output types and values more readable:

Input Type | Input | Result Type | Result
-- | -- | -- | --
java.lang.String | example String | [B | [B@31749b7a
java.lang.String |   | [B | [B@4b469e26
  | null |   | null
java.lang.Integer | 1 |   | ClassCastException: java.lang.Integer cannot be cast to java.lang.String